cmd/deploy.go

```diff
@@ -216,6 +222,34 @@ func newCredentialsCallback() func(registry string) (docker.Credentials, error)
 	}
 }
 
+func newChooseHelperCallback() docker.ChooseCredentialHelperCallback {
+	return func(availableHelpers []string) (string, error) {
+		if len(availableHelpers) < 1 {
+			fmt.Fprintf(os.Stderr, `There is no credential helper available.
+As a result credentials won't be saved.
+To fix this consider installing docker credential helper https://github.com/docker/docker-credential-helpers.
```

Comment:
When does the user experience this? IIUC, the use case would be if they do not have auth credentials in a typical docker config location, so we need to interactively prompt for credentials. Is that right?

If that's right, maybe we can just state it as a recommendation vs. what almost sounds like a warning. 

```suggestion
			fmt.Fprintf(os.Stderr, `Credentials will not be saved.
If you would like to save your credentials in the future,
you can install docker credential helper https://github.com/docker/docker-credential-helpers.
```